pagination/templates/includes/_pagination.html

```diff
@@ -0,0 +1,15 @@
+{% load querystrings %}
+
+{% if paginator.page_range|length > 1 %}
+    <div class="pagination {{ border_class|default:"bordered" }}">
+        {% if page_obj.has_previous %}
+            <a class="prev" href="?{% querystring_update request page=page_obj.previous_page_number %}">Previous</a>
+        {% endif %}
+        {% for pagination_page in paginator.page_range %}
+            <a class="{% if page_obj.number == pagination_page %}selected{% endif %}" href="?{% querystring_update request page=pagination_page %}">{{ pagination_page }}</a>
```

Comment:
We do it all the time.  It's a no-op for the browser, and makes the code cleaner and easier to update (you don't have to move the `{% if %}` inside the `class=""` when you add a second class to it).